integration/yarn/src/test/java/alluxio/yarn/ApplicationMasterTest.java

```diff
@@ -136,39 +102,34 @@ public void after() {
   }
 
   private void setupApplicationMaster(Map<String, String> properties) throws Exception {
+    // This is needed for when the ApplicationMaster calls hadoop's FileSystem.get().
+    System.setProperty("HADOOP_USER_NAME", "testuser");
```

Comment:
do we need to reset this after this integration tests to avoid polluting the system properties of following tests